tests/driver_ina3221/main.c

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the INA3221 sensor driver
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ *
+ * @}
+ */
+
+#include <periph/gpio.h>
+#include <xtimer.h>
+#include "string.h"
```

Comment:
This one is provided by the standard c lib, it should be `#include <string.h>`.

The difference between `<>` and `""` in the include is that with `""` `#include` will first check the local folder for headers with that name, and then will fall back to the system header paths (e.g. those provided by the standard c lib). With `<>` the search will start directly in the system header paths.

Usign `""` instead of `<>` risks that a same named file in the local folder is accidentally included, when the intention was to include a system header.